pkg/db/fetch/fetch.go

```diff
@@ -183,42 +216,22 @@ func Fetch(opts ...Option) error {
 		return errors.Errorf("unexpected schema version. expected: %d, actual: %d", db.SchemaVersion, meta.SchemaVersion)
 	}
 
+	meta.Digest = &digest
 	meta.Downloaded = func() *time.Time {
 		t := time.Now()
```

Comment:
```suggestion
		t := time.Now().UTC()
```

Sorry that this line is not in this PR.
I think it's better to use UTC here in accordance with last_modified.

Without this change:
```
% bbolt get foo.db "metadata" "db" | jq
{
  "schema_version": 0,
  "created_by": "vuls v0.0.1-alpha.0.20250418035303-727e0a682b7a",
  "last_modified": "2025-04-24T02:08:11.400285932Z",
  "digest": "sha256:3e513d10dca2b40b06f93f79c2d5235f4594396e0c67e20facedb5a7ba66c625",
  "downloaded": "2025-04-24T14:19:55.997240885+09:00"
}
```

After:
```
% bbolt get foo.db "metadata" "db" | jq
{
  "schema_version": 0,
  "created_by": "vuls v0.0.1-alpha.0.20250418035303-727e0a682b7a",
  "last_modified": "2025-04-24T02:08:11.400285932Z",
  "digest": "sha256:3e513d10dca2b40b06f93f79c2d5235f4594396e0c67e20facedb5a7ba66c625",
  "downloaded": "2025-04-24T05:24:40.538157519Z"
}
```

Reply:
fixed: https://github.com/MaineK00n/vuls2/pull/177/commits/acf82178b164ed0f7fc7918b8f38fe33a6cb0132